Sources/.swiftlint.yml

```diff
@@ -0,0 +1,59 @@
+disabled_rules:
+  - large_tuple
+  - multiple_closures_with_trailing_closure
+  - todo
+
+opt_in_rules:
+  - empty_count
+  - convenience_type
+  - empty_string
+  - fatal_error_message
+  - first_where
+  - modifier_order
+  - toggle_bool
+  - identical_operands
+  - joined_default_parameter
+  - unavailable_function
+  - overridden_super_call
+  - explicit_init
+  - redundant_optional_initialization
+  - syntactic_sugar
+  - multiline_arguments
+  - multiline_function_chains
+  - prefer_self_type_over_type_of_self
+  - unused_import
+  - attributes
+
+multiline_arguments:
+  only_enforce_after_first_closure_on_first_line: true
+
+attributes:
+  always_on_same_line: ["@IBAction", "@NSManaged", "@objc"]
+
+trailing_whitespace:
+  ignores_empty_lines: true
```

Comment:
IMHO, disable this.

It's against of what all other languages with standard formatters (go, python, js, dart, etc.) are doing by default.

Reply:
The problem is, Xcode by default adds leading/trailing whitespace to newlines and this makes navigation easier in Xcode. If we enable this, we'll need to instruct all new developers to change their Xcode settings, and create a huge PR that removes all empty line whitespaces. So we've decided to go this route, and use default Xcode settings.

Reply:
I'm an outsider but I see. 

It's a problem without xcode but it's convenience for xcode users, which should be the bulk.

Thanks for the explanation.

> create a huge PR that removes all empty line whitespaces
by the way, IMHO, if it's done solely for formatting and kept separate, not a big deal.